lib/client_side_validations/core_ext/regexp.rb

```diff
@@ -1,18 +1,8 @@
 class Regexp
+  require 'js_regex'
```

Comment:
Maybe, but the current position is more robust.

If you move it, you *might* have to make sure that `client_side_validations.rb` is always loaded before `core_ext/regexp.rb` in production and before `test_core_ext.rb` in tests to protect against const_missing errors (depending on load orders and CI setup).